docs/source/using/troubleshooting.adoc

```diff
@@ -113,6 +113,14 @@ cause passwords to fail.
 Workaround: When creating and entering passwords, wrap the string with
 single quotes in the following format: `<password>`
 
+Note: However, ", $, \`, and \ are still interpreted by the shell,
```

Comment:
- Note format should be NOTE: (asciidoc)
- Remove "However", and change to "The characters" (as a prefix for the list)
- Wrap the symbols with ``